### PR TITLE
Change namespace creation modal to let user define its limits etc

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Change namespace creation modal to let user define its limits etc

**Related issue(s)**

### CODE REVIEW

File: core/src/app/content/namespaces/namespace-create/namespace-create.component.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.y-fd-form-label {
+  display: grid;
+  grid-gap: 1em;
+  grid-template-columns: auto max-content;
```
Comment (parostatkiem) [inline core/src/app/content/namespaces/namespace-create/namespace-create.component.scss]: Well done @dariadomagala 👌

File: core/src/app/content/namespaces/namespace-create/namespace-create.component.ts [hunk 2]
```diff
@@ -5,65 +5,241 @@ import { ModalService, ModalComponent } from 'fundamental-ngx';
...
-      : (this.wrongName = false);
+      ? (this.nameError = !regex.test(this.namespaceName))
+      : (this.nameError = false);
   }
 
-  private refreshContextSwitcher() {
-    window.parent.postMessage({ msg: 'luigi.refresh-context-switcher' }, '*');
+  public validateLimitsRegex(change, name) {
+    //  plain integer or plain integer + k | Ki | M | Mi | G | Gi | T | Ti | P | Pi | E | Ei | m 
+    const regex = /^[-+]?[0-9]*(\.[0-9]*)?(([eE][-+]?[0-9]+(\.[0-9]*)?)?|([MGTPE]i?)|Ki|k|m)?$/;
```
Comment (parostatkiem) [inline core/src/app/content/namespaces/namespace-create/namespace-create.component.ts]: Are negative or `0` values allowed by Kyma?

File: core/src/app/content/namespaces/namespace-create/namespace-create.component.ts [hunk 3]
```diff
@@ -5,65 +5,241 @@ import { ModalService, ModalComponent } from 'fundamental-ngx';
...
+      const istioLabel = this.labels.find(this.findIstioLabel);	
+      if (istioLabel) {	
+        this.labels.splice(this.labels.indexOf(istioLabel), 1)	
+      }	
+    }	
+    const istioLabelArray = ['istio-injection', checked.toString()]	
+    this.labels.push(istioLabelArray.join('='))	
+  }
+  
+  public findIstioLabel(label) {
```
Comment (parostatkiem) [inline core/src/app/content/namespaces/namespace-create/namespace-create.component.ts]: How about `isIstioLabel(label: string):boolean` ?

File: core/src/app/content/namespaces/namespace-create/namespace-create.component.ts [hunk 4]
```diff
@@ -5,65 +5,241 @@ import { ModalService, ModalComponent } from 'fundamental-ngx';
...
+    const key = label.split('=')[0];
+    return key === 'istio-injection'
+  }
+
+  public labelsArrayToObject() {
```
Comment (parostatkiem) [inline core/src/app/content/namespaces/namespace-create/namespace-create.component.ts]: `labelsArrayToObject():object` 😃

File: core/src/app/content/namespaces/namespace-create/namespace-create.component.ts [hunk 5]
```diff
@@ -5,65 +5,241 @@ import { ModalService, ModalComponent } from 'fundamental-ngx';
...
+    const labelsObject = {};
+    if (this.labels && this.labels.length > 0) {
+      this.labels.forEach((label) => {
+        const key = label.split('=')[0];
```
Comment (parostatkiem) [inline core/src/app/content/namespaces/namespace-create/namespace-create.component.ts]:
```suggestion
        const [key, value] = label.split('=');
```

File: core/src/app/content/namespaces/namespace-create/namespace-create.component.ts (hunk 2 above)
Comment (dariadomagala) [reply core/src/app/content/namespaces/namespace-create/namespace-create.component.ts]: 0 - yes
<0 - no
good catch 🙂

**Comment by kwiatekus** [comment]
Please follow up on limitRange API changes:
```
"Unknown argument "limits" on field "createLimitRange" of type "Mutation". Did you mean "limitRange"?
```

**Pull request merged**